src/lib/utilities/route-for-api.ts

```diff
@@ -60,6 +63,10 @@ export function routeForApi(
   route: TaskQueueAPIRoutePath,
   parameters: TaskQueueRouteParameters,
 ): string;
+export function routeForApi(
```

Comment:
Again, if we fold this into the workflows list APIs, then we can basically just omit this.